plugins/modules/vmware_tag.py

```diff
@@ -36,13 +37,23 @@
       - Process of updating tag only allows description change.
       required: False
       default: ''
+      aliases: [ 'description' ]
       type: str
     category_id:
       description:
       - The unique ID generated by vCenter should be used to.
       - User can get this unique ID from facts module.
-      required: True
+      - Required if C(category_name) is not set.
+      required: False
+      type: str
+    category_name:
+      description:
+      - The name of category.
+      - Required if C(category_id) is not set.
+      required: False
+      aliases: [ 'category' ]
       type: str
+      version_added: '3.4.0'
```

Comment:
```suggestion
      version_added: '3.5.0'
```

I've just released 3.4.0 in order to get it into Ansible 7.3.0 which is scheduled in a couple of days.

Reply:
I have merged the version change to the pull request.